magento/inventory
# Eliminate button 'Ship' in orders placed with Store pickup method

### Description (*)


### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2657: Eliminate button 'Ship' in orders placed with Store pickup method

### Manual testing scenarios (*)

1. Create order with store pickup shipping method.
2. Navigate to admin order page.
3. Observe 'Ship' button is absent on order page.

### Questions or comments


### Contribution checklist (*)
